# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/linux24benh/arch-ppc-config.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- linux-2.4.22/arch/ppc/config.in	2003-08-25 13:44:40.000000000 +0200
+++ linux-2.4.22-fixed/arch/ppc/config.in	2003-09-16 23:42:48.000000000 +0200
@@ -222,8 +222,10 @@
 
 source drivers/parport/Config.in
 
-tristate 'Generic /dev/rtc emulation' CONFIG_GEN_RTC
-if [ "$CONFIG_GEN_RTC" = "n" -a "$CONFIG_APUS" != "y" ]; then
+if [ "$CONFIG_PPC_RTC" != "y" ]; then
+  tristate 'Generic /dev/rtc emulation' CONFIG_GEN_RTC
+fi
+if [ "$CONFIG_GEN_RTC" != "y" -a "$CONFIG_APUS" != "y" ]; then
   tristate 'Support for /dev/rtc' CONFIG_PPC_RTC
 fi
 
